lib/animina_web/components/top_navigation_components.ex

```diff
@@ -511,6 +526,16 @@ defmodule AniminaWeb.TopNavigationCompontents do
     String.to_atom(photo.state)
   end
 
+  # We Have This checker so that incase we receive the state as a string , we
+  # still convert it to an atom
+  defp make_sure_user_state_is_atom(user) do
```

Comment:
Do we have to do this EVERYWHERE now? If yes, it would make more sense to not use the atom but always to use the string. Have you ask in the Elixirforum about this behaviour?